domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/AppConfigBasedAmPersistenceSwitchTest.java

```diff
@@ -0,0 +1,150 @@
+package uk.gov.hmcts.ccd.definition.store.domain;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static uk.gov.hmcts.ccd.definition.store.domain.AmPersistenceReadSource.FROM_AM;
+import static uk.gov.hmcts.ccd.definition.store.domain.AmPersistenceReadSource.FROM_CCD;
+import static uk.gov.hmcts.ccd.definition.store.domain.AmPersistenceWriteDestination.TO_AM;
+import static uk.gov.hmcts.ccd.definition.store.domain.AmPersistenceWriteDestination.TO_BOTH;
+import static uk.gov.hmcts.ccd.definition.store.domain.AmPersistenceWriteDestination.TO_CCD;
+
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Can we comply with out convention and use Junit5 here?

Reply:
Done